river-schema.yml

```diff
@@ -3433,6 +3433,27 @@ components:
       - id
       - name
       - schema
+    OperationOutcomeIssue:
+      type: object
+      properties:
+        severity:
+          type: string
+        code:
+          type: string
+        diagnostics:
+          type: string
+        location:
+          type: array
+          items:
+            type: string
+        expression:
+          type: string
+      required:
+      - code
+      - diagnostics
+      - expression
+      - location
+      - severity
```

Comment:
I don't know why this is here. From v4 branch, `OperationOutcomeIssue` type should already be described at line 3396 : 

```yml
OperationOutcomeIssue:
      type: object
      properties:
        severity:
          $ref: '#/components/schemas/SeverityEnum'
        code:
          type: string
        diagnostics:
          type: string
        location:
          type: array
          items:
            type: string
        expression:
          type: string
      required:
      - code
      - diagnostics
      - expression
      - location
      - severity
```